ecs-agent/tmds/handlers/fault/v1/handlers/handlers_test.go

```diff
@@ -1450,186 +1365,106 @@ func generateCheckNetworkLatencyTestCases() []networkFaultInjectionTestCase {
 	return append(tcs, commonTcs...)
 }
 
-func TestStartNetworkLatency(t *testing.T) {
-	tcs := generateStartNetworkLatencyTestCases()
-	testNetworkFaultInjectionCommon(t, tcs, NetworkFaultPath(types.LatencyFaultType, types.StartNetworkFaultPostfix))
-}
-
-func TestStopNetworkLatency(t *testing.T) {
-	tcs := generateStopNetworkLatencyTestCases()
-	testNetworkFaultInjectionCommon(t, tcs, NetworkFaultPath(types.LatencyFaultType, types.StopNetworkFaultPostfix))
-}
-
-func TestCheckNetworkLatency(t *testing.T) {
-	tcs := generateCheckNetworkLatencyTestCases()
-	testNetworkFaultInjectionCommon(t, tcs, NetworkFaultPath(types.LatencyFaultType, types.CheckNetworkFaultPostfix))
-}
-
-func generateCommonNetworkPacketLossTestCases(name string) []networkFaultInjectionTestCase {
+func generateCheckNetworkLatencyTestCases() []networkFaultInjectionTestCase {
+	commonTcs := generateCommonNetworkLatencyTestCases(checkNetworkLatencyTestPrefix)
 	tcs := []networkFaultInjectionTestCase{
 		{
-			name:                 fmt.Sprintf("%s no request body", name),
-			expectedStatusCode:   400,
+			name:                 "no-existing-fault-empty-request-payload",
```

Comment:
nit (non-blocking): Thanks for adding this edge case, since we're now going with having no request body as the norm could you also update the tests in `agent/handlers/task_server_setup_test.go`?

https://github.com/aws/amazon-ecs-agent/blob/d904830336ef5112d27188e503bf6e2c3a4f6cda/agent/handlers/task_server_setup_test.go


This can also be done in a follow up PR.

Reply:
SGTM. I'm working on the next PR to clean up the response body to ensure that we don't leak any internal error messages to the customer in the http response. Will make this change in the next PR as well.